src/everwillow/statelib/meta.py

```
@@ -0,0 +1,98 @@
+    ) -> State[V]:
+        """Reconstruct a ``State`` from flattened values.
+
+        Args:
+            values: Flattened values corresponding to the original pytree.
+
+        Returns:
+            Reconstructed ``State`` object.
+        """
+        # can't reconstruct pytree with treedef=None
+        if self.treedef is None:
+            raise ValueError("Cannot reconstruct pytree with treedef=None")
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)